README.md

```diff
@@ -6,7 +6,7 @@
 
 * **Maintainer status:** maintained
 * **Maintainer:** Fernando González Ramos <fergonzaramos@yahoo.es>
-* **Author:** Francisco Martín Rico <fmrico@gmail.com>; Fernando González Ramos <fergonzaramos@yahoo.es>
+* **Author:** Fernando González Ramos <fergonzaramos@yahoo.es>
```

Comment:
Fernando, can you restore my authorship, please?

Reply:
Of course! In this last PR, it is fixed!